File: C/从语言程序设计（苏小红）/平闰年判断.cpp
#include<stdio.h>
main()
{
	int year;
	printf("Input a number:");
	scanf("%d",&year);
	if(year%4==0&&year%100!=0||year%400==0)
	printf("今年是闰年\n");
	else
	printf("今年是平年\n");
}

File: Python/平闰年判断.py
m=int(input("请输入一个年份："))

if m<=0:
   print("输入有误！")
    
if m%400==0 or (m%100!=0 and m%4==0):
   print("输入的年份为闰年")
else:
   print("输入的年份为平年")
   
